java/compaction-status-store/src/test/java/sleeper/compaction/status/testutils/DynamoDBCompactionJobStatusStoreTestBase.java

```diff
@@ -52,8 +52,9 @@
 
 public class DynamoDBCompactionJobStatusStoreTestBase extends DynamoDBTestBase {
 
-    protected static final RecursiveComparisonConfiguration IGNORE_UPDATE_TIMES = RecursiveComparisonConfiguration.builder()
-            .withIgnoredFields("createdStatus.updateTime", "startedStatus.updateTime", "finishedStatus.updateTime", "expiryDate").build();
+    protected static final RecursiveComparisonConfiguration IGNORE_UPDATE_TIMES_AND_TASK_ID = RecursiveComparisonConfiguration.builder()
+            .withIgnoredFields("createdStatus.updateTime", "startedStatus.updateTime",
+                    "finishedStatus.updateTime", "expiryDate", "startedStatus.taskId", "finishedStatus.taskId").build();
```

Comment:
We do want to test the task ID don't we? The only reason the update times are ignored is because we're not setting them, they're generated on the fly. I didn't think it was worth adding a mechanism to set it explicitly. For the task ID we have full control over it.